app/konnect/gateway-manager/data-plane-nodes/upgrade.md

```diff
@@ -16,10 +16,64 @@ We recommend running one major version (2.x or 3.x) of a data plane node per con
 Read through the [{{site.base_gateway}} upgrade considerations](/gateway/latest/upgrade/) for the version that you're upgrading to.
 
 ## Upgrade a data plane node
-
 To upgrade a data plane node to a new version, follow these steps:
 
-1. Open {% konnect_icon runtimes %} **Gateway Manager**, choose a control plane,
+{% navtabs %}
+{% navtab Dedicated Cloud Gateways %}
+{% navtabs %}
+{% navtab Konnect UI %}
+1. Open [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), choose the Dedicated Cloud Gateways powered control plane,
+1. From the **Actions** drop-down menu, select **Update Cluster Config**
+1. Select the desired {{site.base_gateway}} version, and **Update Cluster**.
+
+{{site.konnect_short_name}} will upgrade the version of all the data plane nodes to the version you selected automatically. 
+{% endnavtab %}
+{% navtab API %}
+
+Using the API, you can upgrade your data plane node versions by issuing a POST request to the `/cloud-gateways/configurations` endpoint: 
+
+
+1. Open [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), choose the Dedicated Cloud Gateways powered control plane, and copy the `control_plane_id`.
```

Comment:
```suggestion
1. In [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), select the Dedicated Cloud Gateways powered control plane and copy the `control_plane_id`.
```
Would it be better to use the https://docs.konghq.com/konnect/api/control-planes/latest/#/Control%20Planes/list-control-planes endpoint to retrieve the control plane ID from the API? Just so that all of the steps stay in the API.